apis/v1beta1/httproute_types.go

```diff
@@ -158,12 +158,13 @@ type HTTPRouteRule struct {
 	// Proxy or Load Balancer routing configuration generated from HTTPRoutes
 	// MUST prioritize matches based on the following criteria, continuing on
 	// ties. Across all rules specified on applicable Routes, precedence must be
-	// given to the match with the largest number of:
+	// given to the rule having:
 	//
-	// * Characters in a matching "Exact" path match
```

Comment:
Please pay attention to the rewrite here -- I felt the "number of characters in exact path match" may not have been entirely appropriate. Please do point out if that had some relevance, which I'm missing, and I'll correct the statements.

Reply:
I think that this re-writing related to the exact path match is more accurate 👍

Reply:
The reason for the "number of characters" part is to handle the case of two rules with Exact path matching - so the longer one will win in that case.

Reply:
Ah right, I think I missed that nuance. I withdraw my objection. 😄